app/i18n/locales/ko/about-page.json

```
@@ -0,0 +1,31 @@
+{
+  "title": "우리 팀에 대하여",
+  "description": "PSE는 암호화 기초 구성요소에 대한 사용 사례를 시험하는데 전념하는 이더리움 재단 지원 연구 및 개발 연구소입니다. 우리는 오픈 소스 인프라와 도구를 구축하고, 워크숍과 여름 프로그램을 주최하며, 암호화에 생명을 불어넣는 사람들과 협력합니다.",
+  "our-principles-title": "우리의 원칙",
+  "principles": [
+    {
+      "title": "01. 사람들을 위한 암호화",
+      "description": [
+        "암호화는 어디에나 있습니다. 보안 사이트에 연결하거나 비밀번호로 로그인하거나 휴대폰을 잠금 해제할 때마다 암호화가 작동하는 모습을 볼 수 있습니다. 영지식 증명, 다자간 계산, 동형 암호화와 같은 “프로그래밍 가능한” 암호화를 사용하면 정보 자체를 공개하지 않고도 비밀 정보에 대해 검증 가능한 요청을 수행할 수 있습니다. 이는 신원 관리, 충돌 저항, 익명 커뮤니케이션 등에 적용될 수 있습니다. 우리는 누구나 무료로 사용할 수 있는 오픈 소스 개발 도구, 연구 논문, 프로토타입 라이브러리를 구축하고 있습니다. 우리의 자원들이 사람들이 지역 사회에 필요한 기술을 혁신하도록 영감을 주기를 바랍니다."
+      ]
+    },
+    {
+      "title": "02. 개인정보 우선화",
+      "description": [
+        "우리는 개인정보 보호가 기본적인 권리라고 믿습니다. 우리는 데이터 침해적인 관행에서 벗어나 자신의 개인 정보에 접근할 수 있는 사람이 누구인지에 대한 실질적인 선택권을 사람들에게 제공하는 인터넷을 구축하는데 기여하고자 합니다. 권한은 목적이 구체적이고, 취소 가능하고, 정보에 입각하고, 강제되지 않아야 합니다. 우리는 사람들이 자신을 안전하게 인증하고, 블록체인에서 기밀 거래를 하고, 사용자 개인 정보를 존중하고 보존하는 데 도움이 되는 도구를 만듭니다."
+      ]
+    },
+    {
+      "title": "03. 커뮤니티를 위한 확장",
+      "description": [
+        "영지식 증명은 빠르고 저렴하게 계산을 검증할 수 있어 이더리움과 같은 분산형 시스템이 더욱 효율적이도록 돕습니다. 우리는 더욱 강력하고 실용적인 디지털 공공 인프라에 기여하기 위해 누구나 사용할 수 있는 확장 솔루션을 연구, 설계 및 공유합니다. 우리는 또한 다음세대의 구축자들을 지원하여 커뮤니티를 성장시킵니다. 우리는 암호화를 사용하여 구축하는 데 관심이 있는 모든 사람에게 열려 있는 학생을 위한 몰입감 있는 프로그램과 강의 시리즈를 주최합니다."
```

I suggest using the Korean word "빌더" (a direct transliteration of the English word "builder") instead of "구축자" to maintain the intended meaning. also, suggest to use "개발" instead of "구축하는 데"

---

I think your suggestion for the 'builder' is good. But I think, for the second one, '구축' is more appropriate than '개발' as translation for 'building.' Because I think this service is closer to '구축(build)' the infrastructures than '개발(develop)'. That's why I chose to translate is as '구축'. I look forward for your opinion :)
```suggestion
        "영지식 증명은 빠르고 저렴하게 계산을 검증할 수 있어 이더리움과 같은 분산형 시스템이 더욱 효율적이도록 돕습니다. 우리는 더욱 강력하고 실용적인 디지털 공공 인프라에 기여하기 위해 누구나 사용할 수 있는 확장 솔루션을 연구, 설계 및 공유합니다. 우리는 또한 다음세대의 빌더들을 지원함으로써 커뮤니티를 성장시킵니다. 우리는 암호학을 사용하여 구축하는 데 관심이 있는 모든 사람에게 열려 있는 학생을 위한 몰입감 있는 프로그램과 강의 시리즈를 주최합니다."
```